homeassistant/components/braviatv/media_player.py

```diff
@@ -178,8 +178,10 @@ def __init__(self, host, mac, name, pin):
         self._min_volume = None
         self._max_volume = None
         self._volume = None
+        self._unique_id = None
 
         self._braviarc.connect(pin, CLIENTID_PREFIX, NICKNAME)
+        self._unique_id = self._braviarc.get_system_info().get("cid").lower()
```

Comment:
If we know the key is in the dict we should use `dict[key]`.
```suggestion
        self._unique_id = self._braviarc.get_system_info()["cid"].lower()
```

Reply:
Does `get_system_info` return a cached result?

Reply:
No, it's not cashed.

Reply:
We don't want side effects in init method. Please move this outside init and pass the unique_id as an argument to the entity instead.

Probably create the client outside first too and pass that in too.